### PR TITLE
Modified alpha_k2 

### PR DESCRIPTION
**Pull request by JayeshMD**
The previous implementation of &alpha;<sub>k</sub><sup>2</sup> uses following expression proposed by L. C. Uzal 2011.

The value of  &alpha;<sub>k</sub><sup>2</sup> computed using this expression continues to grow with the length of time series N. Thus, the value of  L<sub>k</sub> does not converge on increasing N, instead L<sub>k</sub> continuously drifts as N is changed. Therefore we can't compare L<sub>k</sub> computed for two reconstructions if N is different for them. To remove the effect of N, the following expression for  &alpha;<sub>k</sub><sup>2</sup> is proposed.

The convergence of L<sub>k</sub> for the Lorenz system, Rössler system, and Mackey-Glass system with N is shown in the following figures. 

Slide2
Slide3
Slide4

In conclusion, N should we considered for L<sub>k</sub> calculation to avoid drift in L<sub>k</sub> caused because of length of time series.

### CODE REVIEW

**Comment by hkraemer** [comment]
Thanks @JayeshMD ,
it is also important to note that your proposed correction actually follows from Eqs. (17), (19) and (20) directly.
I will go through the changes soon.
Cheers

**Comment by hkraemer** [comment]
Hey @JayeshMD ,
unfortunately I can not see what changes you made in the `.mlx`-livescript. What changed? Could you quickly ellaborate on this?

**Comment by JayeshMD** [comment]
Hey @hkraemer,
There is no change in .mlx-LiveScript, it is just the new file with outputs using modified alpha_k. I didn't find any difference in the output. Also, I checked the output of './tests/test_pecuzal.m' which remained the same. 

I think, the reason behind this is, 'uzal_cost_pecuzal.m' used to compute L_decrease is obtained from (L_trial - L) which leads to canceling of log(N) as the time-series considered are of the same length and gave the same results as before.
